runtime/cudaq/operators.h

```diff
@@ -1019,15 +1715,45 @@ class rydberg_hamiltonian {
 };
 
 // type aliases for convenience
+/// @brief Typedef for a map of parameters.
+/// This typedef defines `parameter_map` as a map of strings to complex
+/// numbers, which is used to store the parameters for the operators.
 typedef std::unordered_map<std::string, std::complex<double>> parameter_map;
+/// @brief Typedef for a map of dimensions.
+/// This typedef defines `dimension_map` as a map of integers to 64-bit
+/// integers, which is used to store the dimensions of the operators.
```

Comment:
The dimension is associated with a degree, not an operator.
```suggestion
/// integers, which defines the number of levels (i.e., the dimension) for each degree of freedom an operator targets.
```